docker-compose.yaml

```diff
@@ -32,6 +32,8 @@ services:
     command: npx nx serve backend
     environment:
       - DATABASE_URL=postgresql://dnd-assistant:dnd-assistant@postgres:5432/dnd-assistant?schema=public
+    env_file:
+      - ./.env
```

Comment:
we have env file? if so can update example env file to show necessary keys?

Reply:
We do indeed! It was added in PR #17 

See https://github.com/dnd-assistant/dnd-assistant/blob/main/.example.env